packages/types/src/retry.ts

```diff
@@ -2,11 +2,16 @@ import { Event } from './event';
 import { Response } from './response';
 
 /** Layer used to send data to Amplitude while retrying throttled events in the right order.  */
-export interface RetryClass {
+export interface Retry {
```

Comment:
We should remove `RetryClass` references from README.md

https://github1s.com/amplitude/Amplitude-Node/blob/273/retry-override/packages/node/README.md

Reply:
makes a lot of sense, ty! will change